# line extraction parameters.
m_least_thresh: 0.06 # 3 sigma rule.
m_min_length: 0.45
m_predict_distance: 0.06
m_seed_pts_num: 4
m_min_line_points_num: 5
m_gap_len_thresh: 0.0

# laser scanner parameters.
m_laser_points_num: 180
m_laser_angle_min: -1.5707968705
m_laser_angle_max: 1.5707968705
m_laser_angle_incre: 0.01745329856
m_laser_range_min: 0.0
m_laser_range_max: 10.0

# matching parameters.
plane_pairing_thresh: 1.0
plane_pairing_intersection_ratio: 0.5

hist_bin_width: 2.0
hist_bin_min: -90.0
hist_bin_max: 90.0
hist_search_window: 70
hist_absorb_bin_num: 3
plane_similar_ratio: 5.0
